govuk_navigation_helpers.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "gds-api-adapters", ">= 43.0"
+  spec.add_runtime_dependency "activesupport"
```

Comment:
(It's a little upsetting that we're adding `activesupport` just for improved `Time` functions :disappointed:)